### PR TITLE
cleanup TODOs from PrintFlags wiring

### PR DESCRIPTION
**Pull request by juanvallejo**
**Release note**:
```release-note
NONE
```

Address TODOs from PrintFlags wiring

cc @soltysh @deads2k

### CODE REVIEW

File: pkg/kubectl/cmd/create/create_clusterrolebinding.go [hunk 1]
```diff
@@ -90,10 +90,10 @@ func (o *ClusterRoleBindingOpts) Complete(cmd *cobra.Command, args []string) err
 		return errUnsupportedGenerator(cmd, generatorName)
 	}
 
-	return o.CreateSubcommandOptions.Complete(cmd, args, generator)
+	return o.CreateSubcommandOptions.Complete(f, cmd, args, generator)
 }
 
 // CreateClusterRoleBinding is the implementation of the create clusterrolebinding command.
 func (o *ClusterRoleBindingOpts) Run(f cmdutil.Factory) error {
```
Comment (soltysh) [inline pkg/kubectl/cmd/create/create_clusterrolebinding.go]: Remove the factory.

File: pkg/kubectl/cmd/create/create_pdb.go [hunk 2]
```diff
@@ -100,10 +100,10 @@ func (o *PodDisruptionBudgetOpts) Complete(cmd *cobra.Command, args []string) er
 		return errUnsupportedGenerator(cmd, generatorName)
 	}
 
-	return o.CreateSubcommandOptions.Complete(cmd, args, generator)
+	return o.CreateSubcommandOptions.Complete(f, cmd, args, generator)
 }
 
 // CreatePodDisruptionBudget implements the behavior to run the create pdb command.
 func (o *PodDisruptionBudgetOpts) Run(f cmdutil.Factory) error {
```
Comment (soltysh) [inline pkg/kubectl/cmd/create/create_pdb.go]: This applies to all `create_*` commands.

**Review by soltysh — CHANGES_REQUESTED** [review]

**Comment by juanvallejo** [comment]
@soltysh thanks, removed factory from Run method in create cmds

**Review by soltysh — APPROVED** [review]
/lgtm
/approve no-issue

**Pull request merged**